### ISSUE TITLE
Virtual attributes ignored when association exists

### ISSUE THREAD

**Issue by seanlinsley** [activeadmin/activeadmin#4920]
Note: the below references CSV export but it's the index page that is exhibiting this bug.

------

Previously given SQL to add a virtual attribute to a CSV export, then referencing it like this:
```ruby
csv do
  column :association_id
```
`association_id` would be directly included in the CSV. After upgrading from 5536b158bf0338ea54ff to 1.0.0.pre5 on Rubygems, `DisplayHelper` now tries to call the association method on the model, instead of the `association_id` virtual attribute.

https://github.com/activeadmin/activeadmin/blob/v1.0.0.pre5/lib/active_admin/view_helpers/display_helper.rb#L58

**Comment by seanlinsley** [comment]
Correction: this affects the index page, defined like this:
```ruby
index do
  column :association_id
```
It appears that CSVs continue to work the same way. I'll update the ticket name.

**Comment by Fivell** [comment]
@seanlinsley ,  can you please check if fixed by https://github.com/activeadmin/activeadmin/pull/4784/commits/1ef08af5044814c336917fa93aea607dce16dcb7 ?

**Comment by deivid-rodriguez** [comment]
Yes, the commit pointed to by @Fivell might fix it.

I'm not sure I understand what you mean, though. I usually refer to as "virtual attributes" to attributes in the model with no corresponding column in DB, but in this case, since you mention that the association is defined, I would expected the column in DB to be defined, acting as a foreign key.

**Comment by seanlinsley** [comment]
It's defined on the other side of the association; it's `has_one` instead of `belongs_to`.

**Comment by Fivell** [comment]
@seanlinsley , so what about 1.0.0? still doesn't work?

**Comment by deivid-rodriguez** [comment]
Ok, so you have something like

```
# app/models/user.rb
# column "name" defined on "User"
class  User < ApplicationRecord
  has_one :profile
end

# app/models/profile.rb
# column "user_id" defined in "profiles" table
class Profile < ApplicationRecord
  belongs_to :user
end
```

And when you configure

```
index do
  column :user_id
end
```

it prints the user's name.

Is that it? If that's the behavior, I think it's a bug.

**Comment by seanlinsley** [comment]
Sorry for the delay. This is fixed in 1.0.0.
